.github/workflows/tests_gpu_cu11.yml

```
@@ -183,14 +183,19 @@ jobs:
           fetch-tags: true
           path: main
 
+      - name: Switch to release build of Lightning
+        if: inputs.lightning-version == 'release'
+        run: |
+          cd main
+          git fetch --all
+          git checkout $(git branch -a --list origin/v*rc* | tail -1)
```

These work with bash, but not with zsh. We can fix that by putting quotes here, and guaranteeing it works on all shells (e.g. if we add MacOS tests here, we need zsh support).

```suggestion
          git checkout $(git branch -a --list "origin/v*rc*" | tail -1)
```